src/server/controllers/public-api/index.ts

```diff
@@ -0,0 +1,107 @@
+import type {Request, Response} from '@gravity-ui/expresskit';
+import {AppError, REQUEST_ID_PARAM_NAME} from '@gravity-ui/nodekit';
+import _ from 'lodash';
```

Comment:
We don't use `_` for whole import of `lodash`
We use `import func from 'lodash/func'` instead

PS I don't see any usage of it